README.md

```diff
@@ -81,6 +81,7 @@ Apio is used by [Icestudio](https://github.com/FPGAwars/icestudio).
 | Board name | Interface |
 |:-|:-:|
 | [TinyFPGA B2](http://tinyfpga.com/b-series-guide.html) | Serial |
+| [TinyFPGA BX](http://tinyfpga.com/b-series-guide.html) | Serial |
```

Comment:
Just to be sure, don't you want to use other URL, like `https://github.com/tinyfpga/TinyFPGA-Bootloader/tree/master/programmer` for example?

Reply:
The b-series-guide will be updated for both B2 and BX boards, so I'm happy to use this URL for now.   Any URL change that I might want later will be low-priority.